packages/app-project/src/shared/components/NavLink/NavLink.js

```diff
@@ -43,7 +43,7 @@ function NavLink ({
 }
 
 NavLink.propTypes = {
-  color: PropTypes.string,
+  // color: PropTypes.string, can also be Grommet object {{ light, dark }}
```

Comment:
Constraining `color` to a `string` doesn't make sense because with Grommet you can do `{ light: 'brand', dark: 'white'}. Might need to see if Grommet uses propTypes and how they define `color`'s type.